### DISCUSSION TITLE
Questions about the parameters of troposphere estimation? The results show 12 parameters

### DISCUSSION THREAD

**Discussion by FangXinHu** [groops-devs/groops#125] [Q&A] [answered]
According to the sample of PPP strategy ,the troposphere estimation parameters column include : MJD, ZHD, ZWD, dry north gradient, wet north gradient, dry east gradient, wet east gradient;But the results show 12 columns.How to understand these factors belong to the estimation? where are the parameters of residual gradient delays(1 degree spline of a day) of north and east ( They are supposed to be four unchanged parameters  a day )?

**Comment by FangXinHu** [comment]
Thanks for your suggestions!

**Comment by Waley04** [comment]
Hi @FangXinHu,

I also noted the same. Clarification about this will assist.

Thanks.

**Comment by tmayerguerr** [comment, marked as answer]
In the documentation only the first relevant columns are described.
The file is a time series (defined by the timeSeries of GnssProcessing) with the evaluated parameters and model values.
So, not the parameters itself, but as final values.
The complete list per epoch is
- time (MJD)
- tropospheric zenith dry delay [m] (only from model)
- tropospheric zenith wet delay [m] (model + delta estimate)
- tropospheric dry gradient - north direction [m] (model + delta estimate, due to same mapping function)
- tropospheric wet gradient - north direction [m] (only from model)
- tropospheric dry gradient - east component [m] (model + delta estimate, due to same mapping function)
- tropospheric wet gradient - east component [m] (only from model)
- tropospheric zenith wet delay [m] (delta estimate)
- tropospheric gradient - north [m] (delta estimate)
- tropospheric gradient - east  [m] (delta estimate)
- dry mapping function coefficient a []
- wet mapping function coefficient a []

regards
Torsten

**Reply by FangXinHu** [reply]
Sincerely thanks！it helps a lot!

**Reply by FangXinHu** [reply]
Hello developer, I would like to ask for more details. Does the third item of the output include the eighth item?Similarly, the fourth term of the output includes the ninth term， the sixth term of the output includes the tenth term? Delta estimate are already included in it? Thank you for your answers.

**Reply by FFFFFG-IGG** [reply]
> In the documentation only the first relevant columns are described. The file is a time series (defined by the timeSeries of GnssProcessing) with the evaluated parameters and model values. So, not the parameters itself, but as final values. The complete list per epoch is
> 
> * time (MJD)
> * tropospheric zenith dry delay [m] (only from model)
> * tropospheric zenith wet delay [m] (model + delta estimate)
> * tropospheric dry gradient - north direction [m] (model + delta estimate, due to same mapping function)
> * tropospheric wet gradient - north direction [m] (only from model)
> * tropospheric dry gradient - east component [m] (model + delta estimate, due to same mapping function)
> * tropospheric wet gradient - east component [m] (only from model)
> * tropospheric zenith wet delay [m] (delta estimate)
> * tropospheric gradient - north [m] (delta estimate)
> * tropospheric gradient - east  [m] (delta estimate)
> * dry mapping function coefficient a []
> * wet mapping function coefficient a []
> 
> regards Torsten

Hello Torsten! Parameters you've detailed above seems not consistent with what Sebastian Strasser mentioned in his PhD thesis. Troposphere parameters in Strasser thesis:
![image](https://user-images.githubusercontent.com/59271400/210042095-a9e5c9ee-0e38-401d-b879-66771d0677e6.png)
It's shown that dry/wet/residual gradient share different mapping function. But what you've mentions above denotes residual gradient 
 share the same funciton with dry gradient? Which one should we use to calculate total troposhpere delay?If possible, could you please tell us how to use the output parameters to calculate total slant and zenith troposphere delay?
Thanks in advance!

**Reply by tmayerguerr** [reply]
We use the dry gradient mapping function for the estimation of residual gradients.
regards
Torsten

**Reply by FFFFFG-IGG** [reply]
Hi Torsten，
  I‘m not sure how to get the total ztd according to what you have shown, I sum the 
1.tropospheric zenith dry delay [m] (only from model)
2.tropospheric zenith wet delay [m] (model + delta estimate)
3.tropospheric dry gradient - north direction [m] (model + delta estimate, due to same mapping function)
4.tropospheric wet gradient - north direction [m] (only from model)
5.tropospheric dry gradient - east component [m] (model + delta estimate, due to same mapping function)
6.tropospheric wet gradient - east component [m] (only from model)
and compare the rusults with IGS ZTD and found a large RMSE when I used GPS L1/L5 or L2/L5, could you please tell me if I did thing right and whether the result normal  when using L1/L5 or L2/L5 
regards,
Yongzhao Fan
